library/kotlin/src/io/envoyproxy/envoymobile/Client.kt

```diff
@@ -18,15 +20,19 @@ interface Client {
    * @param body Serialized data to send as the body of the request.
    * @param trailers Trailers to send with the request.
    * @param responseHandler the callback for receiving stream events.
+   * @return CancellableStream, a cancelable request.
```

Comment:
nit: Here and everywhere else in the diff should be `CancelableStream` for parity with iOS and the rest of the spellings